src/js/stores/AppsStore.js

```diff
@@ -230,7 +246,7 @@ QueueStore.on(QueueEvents.CHANGE, function () {
 AppDispatcher.register(function (action) {
   switch (action.actionType) {
     case AppsEvents.REQUEST_APPS:
-      storeData.apps = processApps(action.data.body.apps);
+      storeData.apps = processAppsAndGroups(action.data.body);
```

Comment:
We should rename this property (as well at the getter) as it include apps and groups. Maybe we should provide different getters, one which returns all items, one for apps and one for groups.